### PR TITLE
Implemented kueue tests 

### PR DESCRIPTION
**Pull request by ighosh98**
### Submission Checklist
Added TAS and general kueue tests. Used a2-high as the availability of nodes would be higher.

NOTE: Community submissions can take up to 2 weeks to be reviewed.

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cluster Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: modules/management/kubectl-apply/variables.tf [hunk 1]
```diff
@@ -55,12 +55,11 @@ variable "kueue" {
   description = "Install and configure [Kueue](https://kueue.sigs.k8s.io/docs/overview/) workload scheduler. A configuration yaml/template file can be provided with config_path to be applied right after kueue installation. If a template file provided, its variables can be set to config_template_vars."
   type = object({
     install              = optional(bool, false)
-    version              = optional(string, "v0.8.1")
+    version              = optional(string, "v0.9.0")
```
Comment (annuay-google) [inline modules/management/kubectl-apply/variables.tf:58]: Do not change the default here?

File: tools/cloud-build/daily-tests/ansible_playbooks/test-validation/test-gke-kueue.yml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+- name: Ensure all pods are on the same host
+  delegate_to: localhost
+  ansible.builtin.shell: |
+    kubectl get pods \
+    -o custom-columns="Name:.metadata.name,Host:.spec.nodeSelector.cloud\.google\.com/gce-topology-host" | \
+    sort -k2 | uniq -f 1 | wc -l
+  register: unique_host_count
+  until: unique_host_count.stdout | int == 2
+  retries: 5
```
Comment (annuay-google) [inline tools/cloud-build/daily-tests/ansible_playbooks/test-validation/test-gke-kueue.yml:53]: Increase retries to 10

File: tools/cloud-build/daily-tests/blueprints/gke-a2-highgpu.yaml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      network_count: 4
+      subnetwork_cidr_suffix: 24
+      mtu: 8244
+
+  - id: gke_cluster
+    source: modules/scheduler/gke-cluster
+    use: [network1, gpunets, gke_service_account]
+    settings:
+      enable_private_endpoint: false  # Allows for access from authorized public IPs
+      release_channel: 'RAPID'
```
Comment (annuay-google) [inline tools/cloud-build/daily-tests/blueprints/gke-a2-highgpu.yaml:70]: remove once Kueue graduates to stable channel (create buganizer item)

**Comment by annuay-google** [comment]
Please monitor tests for flakiness after merging to develop

**Review by annuay-google — APPROVED** [review]
Please fix the minor comments, otherwise LGTM

File: tools/cloud-build/daily-tests/blueprints/gke-a2-highgpu.yaml (hunk 3 above)
Comment (ighosh98) [reply tools/cloud-build/daily-tests/blueprints/gke-a2-highgpu.yaml:70]: Will create backlog item.

**Pull request merged**